frontend/src/components/Layout/Layout.tsx

```diff
@@ -1,49 +1,67 @@
-import { Outlet } from "react-router-dom"; //using outlet so that whatever page user navigates to it gets autofilled to that
+//using outlet so that whatever page user navigates to it gets autofilled to that
+import { Outlet } from "react-router-dom";
 import { useNavigate, useLocation } from "react-router-dom";
+//Navigation Menu component from shadcn: https://ui.shadcn.com/docs/components/navigation-menu
 import {
   NavigationMenu,
   NavigationMenuItem,
   NavigationMenuLink,
   NavigationMenuList,
 } from "@/components/ui/navigation-menu";
 
-
-
 export default function Layout() {
-
   const location = useLocation(); //using location to figure out what path we are currently on to highlight it to user
```

Comment:
I mean, the suggestion was for ALL the comments lol
```suggestion
  // Using location to figure out what path we are currently on to highlight it to user
  const location = useLocation(); 
```

Reply:
OH LOL sorry I misunderstood. I also think the formatter puts the comments on the sides sometimes I dont know it is like wonky. Will keep in mind for future comments